src/i18n/translations/english.json

```diff
@@ -10,11 +10,18 @@
     "browse": "Browse Scholarships",
     "add": "Add a Scholarship",
     "login": "Login",
+    "load": "Load More",
     "subscribe": "SUBSCRIBE FOR UPDATES"
   },
 
   "constructAlert": {
     "title": "Warning",
     "description": "🚧 Website Actively Under-Construction! 🚧"
+  },
+
+  "listScholarships": {
+    "title": "Scholarships",
```

Comment:
You could have a `general.scholarships` or `common.scholarships` key for this so we don't have to repeat this multiple times.